### ISSUE TITLE
must specify script lang attribute when the code format is not standard javascript.

### ISSUE THREAD

**Issue by lovetingyuan** [vuejs/eslint-plugin-vue#1567]
**Please describe what the rule should do:**

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[x] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<script>
export default {
  render(h) { return <div></div> }
}
</script>
```
warn: script is lack of lang attribute, should set lang="jsx"

**Additional context**

**Comment by ota-meshi** [comment]
Thank you for this issue.

I don't think it is necessary to use SFC when using JSX.
Why do you need to use SFC?

**Comment by lovetingyuan** [comment]
> Thank you for this issue.
> 
> I don't think it is necessary to use SFC when using JSX.
> Why do you need to use SFC?

Yes, it is not necessary, just because the current codebase uses jsx in sfc. I think we should better rewrite jsx to template, thanks.
